src/App.js

```diff
@@ -62,10 +69,10 @@ function App() {
       console.error(error);
     }
```

Comment:
if you need to implement a catch you need to do in this block correct way to implement it is .

```
try {
//some logic here
}catch(error){
   console.log(error)
}
```